scheduler/cupsfilter.c

```diff
@@ -235,7 +247,10 @@ main(int  argc,				/* I - Number of command-line args */
 		  srctype = argv[i];
 		}
 		else
-		  usage(opt);
+    {
+      _cupsLangPrintf(stderr,_("%s: Error - expected source MIME type after \"-i\" option."),argv[0]);
+		  usage(NULL);
+    }
```

Comment:
I don't exactly know why its showing like this in github. I am using vscode editor for editing files. `else` statement and opening brace `{` are being shown in vertical line there.